### ISSUE TITLE
Иконки душ

### ISSUE THREAD

**Issue by hippogamesunity** [hippogamesunity/SpellMasters#62]
Прошу подготовить иконки душ воинов.

Все иконки должны быть 256х256 на черном фоне. Предлагаю начать с базы:
- макет иконки лучника 1 профы
- макет иконки воина 1 профы
- макет иконки мага 1 профы
- иконки, скажем, разведчика и охотника будут отличаться только цветом
- иконки 2 и 3 профы должны быть развитием иконок 1-й профы, т.е. быть немного больше, круче, ярче
- иконки должны сочетаться с купленными иконками из платных паков, посмотреть можно в магазине (в игре)

Когда согласуем макеты, нужно раскрасить их и сохранить для каждого юнита:

Список иконок:
- DummySoul
- MilitiamanSoul
- MercenarySoul
- SlaveSoul
- ScoutSoul
- HunterSoul
- FireSchoolAdeptSoul
- WaterSchoolAdeptSoul
- ClericSoul
- HermitSoul
- SpearmanSoul
- GuardSoul
- BanditSoul
- BerserkSoul
- GladiatorSoul
- RangerSoul
- ThiefSoul
- PathfinderSoul
- FireWizardSoul
- FireWarriorSoul
- ExileSoul
- WaterWizardSoul
- BishopSoul
- WarlockSoul
- InquisitorSoul
- ShamanSoul
- HalberdierSoul
- KnightSoul
- PaladinSoul
- AssassinSoul
- MarauderSoul
- DestroyerSoul
- ChampionSoul
- RoyalArcherSoul
- SiegeArcherSoul
- SniperSoul
- WitchHunterSoul
- PyromancerSoul
- FireKnightSoul
- WitchSoul
- NecromancerSoul
- StormBringerSoul
- IceWizardSoul
- CardinalSoul
- DruidSoul

PNG иконки необходимо разместить в SpellMasters\Assets\Images\ItemIcons
PSD исходник 256х256 SpellMasters\Resources\Icons\Souls.psd

Чтобы проверить текущие иконки, достаточно зайти в игровой магазин:
![image](https://user-images.githubusercontent.com/7318172/31576631-592b76c2-b107-11e7-91fb-819949504157.png)

**Comment by hippogamesunity** [comment]
Кстати как тебе такой стиль? Смайлики сочетаются с чибиками и твоими менюшками
![image](https://user-images.githubusercontent.com/7318172/31576878-f6995208-b10c-11e7-81b6-9e6c09784352.png)

**Comment by Ragewortt** [comment]
Стиль нравится. Теперь нужно подумать как зделать, чтобы в них узнавался тот или иной юнит.

**Comment by hippogamesunity** [comment]
По форме должен угадываться тип (лук, меч, маг), а цвет должен быть ближе к эквипу юнита.
Может как-то так?
![image](https://user-images.githubusercontent.com/7318172/31577012-6ff0b27a-b10f-11e7-8aef-3210edd20f79.png)

**Comment by Ragewortt** [comment]
хмм...может сделать им шапки - у воина шлем, у лучника эта штука с пером и т.д. сейчас попробую

**Comment by Ragewortt** [comment]
![1](https://user-images.githubusercontent.com/22120368/31577542-faf1476c-b129-11e7-9564-24251bfa0a0a.png)

**Comment by Ragewortt** [comment]
![2](https://user-images.githubusercontent.com/22120368/31577844-3dc25a68-b12f-11e7-8bb1-85015531cbd4.png)

**Comment by hippogamesunity** [comment]
Вставь в инвентарь, посмотрим как будет смотреться. Мне пока кажется, первый вариант (DarkSouls) 
проще и лучше.

**Comment by Ragewortt** [comment]
добавил еще пару вариантов с шапками.  у них плюс в том, что можно просто менять шапки для каждого юнита. а для нового уровня можно добавить маленькие светящиеся штуки и прочие эффекты.
![ass2](https://user-images.githubusercontent.com/22120368/31586435-69ceb140-b1ea-11e7-9aef-35cc5fafb01d.png)
![screenshot_2017-10-15_20-52-00](https://user-images.githubusercontent.com/22120368/31586446-89c4c93a-b1ea-11e7-8717-cbfe224fffa3.png)

**Comment by hippogamesunity** [comment]
Вот последние два варианта вроде бы ничего. Только сделай черный фон
(совсем).

15 октября 2017 г., 18:55 пользователь Ragewortt <notifications@github.com>
написал:

**Comment by Ragewortt** [comment]
![screenshot_2017-10-15_22-01-50](https://user-images.githubusercontent.com/22120368/31587072-4ae9f136-b1f4-11e7-82c8-2476e73c20ad.png)
![ass2](https://user-images.githubusercontent.com/22120368/31587077-6453c660-b1f4-11e7-9ad8-9d632ae2ee29.png)

**Comment by Ragewortt** [comment]
Как тебе эти варианты для первого уровня. Цвет пока не стал подгонять под каждого юнита, оставил общий.
красный - воины
зеленый - лучники
синий - маги
Единственная проблема, как различать юнитов в капюшонах. 
![screenshot_2017-10-17_03-45-41](https://user-images.githubusercontent.com/22120368/31638612-0c11c27c-b2ee-11e7-9bbf-3f9762db24aa.png)

**Comment by hippogamesunity** [comment]
Четно говоря не очень, особенно в сравнении с другими иконками. Давай
вернемся к первому варианту (dark souls)?

17 октября 2017 г., 1:50 пользователь Ragewortt <notifications@github.com>
написал:

**Comment by Ragewortt** [comment]
Это который самый первый вариант с абстрактных формами?

**Comment by hippogamesunity** [comment]
Да.

2017-10-17 9:58 GMT+03:00 Ragewortt <notifications@github.com>:

**Comment by Ragewortt** [comment]
http://darksouls3.wiki.fextralife.com/page/revision/Boss+Soul+Weapons

**Comment by hippogamesunity** [comment]
И черный фон не забудь. Лучше градацию сделать размером, а типы
юнитов-формой.

**Comment by Ragewortt** [comment]
Сейчас там 10 форм для первых 10 юнитов. Для 2 и 3 профы иконка будет та же только с бОльшим размером души или другим цветом (где необходимо)?
![screenshot_2017-10-18_00-40-21](https://user-images.githubusercontent.com/22120368/31685549-c5e486a4-b39c-11e7-9b84-4789148de7a1.png)

**Comment by hippogamesunity** [comment]
На заднем фоне можно сделать эффекты магии вроде как я для лука сделал
![image](https://user-images.githubusercontent.com/7318172/31685767-d2c36184-b38c-11e7-86b5-9a95ccb4fb77.png)
Я прикопал пару кистей для фотошопа, но можешь и сам поискать:
SpellMasters\Resources\Icons\PhotoshopBrushes

**Comment by Ragewortt** [comment]
для каждой иконки свой фон?

**Comment by Ragewortt** [comment]
При отдалении фон смешивается с душами.
![111](https://user-images.githubusercontent.com/22120368/31687258-2986a188-b3a2-11e7-9404-e94a4bd3cef3.png)

**Comment by Ragewortt** [comment]
Добавил эффекты на фон. как тебе?
![screenshot_2017-10-18_21-50-09](https://user-images.githubusercontent.com/22120368/31731409-68bf7b7a-b44e-11e7-972d-10ca49630767.png)

**Comment by hippogamesunity** [comment]
Да, так намного лучше. Не обязательно делать разные. Выбери самый красивый,
и раскрашивай его вместе с душой. Мне кажется с завихрушкой прилично
смотрится, но можно поверх черного фона добавить еще градиент (см. как на
предметах сделано).

2017-10-18 19:52 GMT+03:00 Ragewortt <notifications@github.com>:

**Comment by Ragewortt** [comment]
с завихрушкой это какой именно?)

**Comment by hippogamesunity** [comment]
Моя магическая кисть

On Oct 18, 2017 22:07, "Ragewortt" <notifications@github.com> wrote:

с завихрушкой это какой именно?)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/hippogamesunity/SpellMasters/issues/62#issuecomment-337696573>,
or mute the thread

**Comment by Ragewortt** [comment]
С градиентом души сливались так что оставил без него. На всякий случай закинул исходник в  SpellMasters\Resources\Icons\Souls.psd
![screenshot_2017-10-19_03-46-49](https://user-images.githubusercontent.com/22120368/31746380-814d8754-b480-11e7-87b1-9b7b1efbcfe0.png)

**Comment by hippogamesunity** [comment]
Ага, я поиграюсь с ним.

2017-10-19 1:53 GMT+03:00 Ragewortt <notifications@github.com>:
